recipes-domd/agl/inc/domd-agl-image.inc

```diff
@@ -41,6 +41,9 @@ configure_versions_rcar() {
 
     # vGPU settings
     base_update_conf_value ${local_conf} PVR_NUM_OSID "2"
+
+    # HACK: force ipk instead of rpm b/c it makes troubles to PVR UM build otherwise
+    base_update_conf_value ${local_conf} PACKAGE_CLASSES "package_ipk"
```

Comment:
What is the AGL default here?
Would it lead to any troubles to other parts of the system?

Reply:
1. default is weak:
PACKAGE_CLASSES ?= "package_rpm"
and AGL doesn't change it.
prod-devel's Poky changes this to "package_rpm".
2.1. It would be good if we can remove this hack, but it does need deeper investigation
2.2. The hack doesn't hurt the rest and is valid choice as well as deb or rpm

Reply:
I mean if something AGL specific relies on rpm and do not work with apk.
Anyway, as you said, we will see it further.

Reply:
yes, the patch is marked as HACK
I will open an issue on that

Reply:
https://github.com/xen-troops/meta-xt-prod-ces2018/issues/4